zipkin/src/test/java/zipkin/storage/SpanStoreTest.java

```diff
@@ -519,11 +520,11 @@ public void getTraces_differentiateOnServiceName() {
 
     Span trace2 = Span.builder().traceId(2).name("get").id(2)
         .timestamp((today + 2) * 1000)
-        .addAnnotation(Annotation.create((today + 1) * 1000, CLIENT_SEND, APP_ENDPOINT))
```

Comment:
this data had incorrectly aligned timestamps (which wasn't the point of the test)